migrations/20161112173153_add-eligibility.js

```diff
@@ -1,7 +1,7 @@
 exports.up = function (knex, Promise) {
   return knex.schema.createTable('Eligibility', function (table) {
     table.integer('EligibilityId').unsigned().primary().unique()
-    table.string('Reference', 10).notNullable().index().unique()
+    table.string('Reference', 10).notNullable().index()
```

Comment:
do you need to add these constraints after all the migrations have run? 

Is there a prod system for this that this could mess with? 

Another approach would be to ditch all the existing migrations, export the existing schema from prod as migration 0 and start again. Would need to reset migrations table etc..

Reply:
I was wondering this as there are numerous tables that have never been added to the migrations hence me working backwards from them. Just a question of converting them to knex.

Reply:
@andrewrlee sorry had to rebase before it'd let me merge so needs another approval 😄